common/i18n/resources/en.js

```diff
@@ -267,6 +281,13 @@ module.exports = {
       analysis: 'Analysis',
     },
     confirmNodeDelete: 'Are you sure you want to delete this item?',
+    dataVisHeader: {
```

Comment:
why adding this new param? can't you just use the prefix that we have? e.g. nodeDefTaxon, nodeDefFile?

Reply:
TableColumns uses `{i18n.t(`surveyForm.dataVisHeader.${NodeDefTable.extractColName(nodeDef, col)}`)}`. How would you change the prefix depending on the context (file, taxon, code)?

Reply:
you can get it from NodeDef.getType and capitalise the first letter

Reply:
Ok